### ISSUE TITLE
Smart copy-paste of source code including imports

### ISSUE THREAD

**Issue by Gama11** [fdorg/flashdevelop#698]
IntelliJ automatically adds the imports for a a copy-pasted code section, which is extremely convenient.

**Comment by elsassph** [comment]
Does IntelliJ include imports when pasting code from another editor?

**Comment by Gama11** [comment]
No. I should also mention that there's a confirmation popup before it actually does the importing:

![](http://i.imgur.com/AO1YwhW.png)

**Comment by elsassph** [comment]
Trying to identify missing imports will be a slightly prohibitive procedure in FD because I believe IntelliJ types all the code all the time (and thus knows what is known and what isn't), while in FD we'd have to resolve every single identifier.
